### DISCUSSION TITLE
Gradient overlay for the line/area section only?

### DISCUSSION THREAD

**Discussion by Doko-Demo-Doa** [Abhinandan-Kushwaha/react-native-gifted-charts#531] [Q&A] [answered]
Hi, thank you for the great library. It is easy to use and very customizable.

However, how can I achieve this? A simple area chart but have linear gradient on the line only:

![image](https://github.com/Abhinandan-Kushwaha/react-native-gifted-charts/assets/7723097/abd2715e-42a9-472f-a477-d0a06702fda4)

Is there a proper props to get this result?

**Comment by Abhinandan-Kushwaha** [comment, marked as answer]
Hi @Doko-Demo-Doa 👋
To apply gradient on the line, you can use the following props-

- lineGradient
- lineGradientDirection
- lineGradientStartColor
- lineGradientEndColor

[See the docs](https://gifted-charts.web.app/linechart/#multicolor)
[See related issue](https://github.com/Abhinandan-Kushwaha/react-native-gifted-charts/issues/251)
